core/templates/dev/head/pages/subtopic_viewer/SubtopicPageInfoService.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to hold info to be displayed on the subtopic page.
+ */
+
+oppia.factory('SubtopicPageInfoService', [
```

Comment:
Why not have subtopic page data also in this? Otherwise the service seems incomplete.
Also tests.